protocol/messenger.go

```diff
@@ -3313,25 +3313,38 @@ func (r *ReceivedMessageState) addNewActivityCenterNotification(publicKey ecdsa.
 	var idToUse string
```

Comment:
notificationID will be more obvious

Reply:
I don't understand its not my code :)

Reply:
I know that code was added by @jrainville if he don't mind I can fix :)

Reply:
You can change it